modules/Makefile

```diff
@@ -0,0 +1,72 @@
+
+SUBDIRS = rejson redistimeseries redisbloom redisearch
```

Comment:
Wrong folder name for ReJSON

Reply:
I suggest replacing the name of the directory to rejson instead of changing it here to `redisjson`